src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: LAMPy stack monitoring integration
+tags:
+    - New Relic integrations
+    - LAMPy
+    - Linux
+    - Apache
+    - Python
+metaDescription: Use New Relics Linux, Apache, Python and MySql monitoring to get a dashboard with metrics from your LAMPy application.
+
+---
+
+import infrastructureLampyDashboard from 'images/infrastructure_screenshot-full_lampy-dashboard.png'
+
+Our LAMPy integration makes use of the Infrastructure agent, MySQL agent, Apache agent, and Python agent to provide you with a pre-built dashboard with your most important metrics like response time, CPU utilization, traffic, and login frequencies.
+
+
+<img
+  title="Lampy dashboard"
+  alt="A screenshot depicting the LAMPy prebuilt dashboard"
+  src={infrastructureLampyDashboard}
+/>
+
+<figcaption>
+After setting up our LAMPy integration, we give you a dashboard for your LAMPy web app metrics. 
+</figcaption>
+
+## Install [#install]
+
+### Step 1: Install the infrastructure agent [#infra-install]
+
+To do this, follow [the infrastructure agent install steps](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) for the host containing your LAMPy application.
+
+### Step 2: Install the MySQL agent [#mysql-install]
+
+Our LAMPy integration relies on the MySQL agent to work. To learn more and check requirements review our [MySQL docs](https://docs.newrelic.com/install/mysql/).
+
+1. From [one.newrelic.com](https://one.newrelic.com) click **Add data ** > Infrastructure & OS  > MySQL**.
+2. Follow the instructions to install the MySQL agent.
+
+### Step 3: Install the Apache agent [#apache-install]
+
+Our LAMPy integration relies on the Apache agent to work. To learn more and check requirements review our [Apache docs](/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/).
+
+1. From [one.newrelic.com](https://one.newrelic.com) click **Add data ** > Infrastructure & OS  > Apache**.
+2. Follow the instructions on the screen to install Apache agent.
+
+### Step 4: Install the Python agent [#python-install]
+
+Our LAMPy integration relies on the Python agent to work. To learn more and check requirements review our [Python docs](/docs/apm/agents/python-agent/getting-started/introduction-new-relic-python/).
+
+1. From [one.newrelic.com](https://one.newrelic.com) click **Add data ** > Application monitoring  > Python**.
+2. Name your application.
+3. Download the configuration file and place it in your application's root directory.
+4. Integrate python agent for Django web site which is running on Apache server 
```

Comment:
Some of these are a little hard to parse.  #4 here is tough. Also 'Python'. Seems like it could use a readability pass as sometimes they're just sentence fragments/incomplete

Reply:
I think I fixed this!